File: Cnote/app/src/main/java/srmicrosystems/cnote/Model/ManifestItem.java
package srmicrosystems.cnote.Model;

/**
 * Created by saman_000 on 11-06-2016.
 */
public class ManifestItem {
    private String CNoteNo;

    private int LoadedQuantity;

    private String ManifestId;

    public String getCNoteNo ()
    {
        return CNoteNo;
    }

    public void setCNoteNo (String CNoteNo)
    {
        this.CNoteNo = CNoteNo;
    }

    public int getLoadedQuantity ()
    {
        return LoadedQuantity;
    }

    public void setLoadedQuantity (int LoadedQuantity)
    {
        this.LoadedQuantity = LoadedQuantity;
    }

    public String getManifestId ()
    {
        return ManifestId;
    }

    public void setManifestId (String ManifestId)
    {
        this.ManifestId = ManifestId;
    }
}

File: Consignment_Track_SSCPL/Consignment_Track_SSCPL/ManifestItem.cs
using System;

namespace Consignment_Track_SSCPL
{
    public class ManifestItem
    {
        public int MItemID { get; set; }
        public string ManifestId { get; set; }
        public string CNoteNo { get; set; }
        public int? LoadedQuantity { get; set; }
        public DateTime? ItemDate { get; set; }
    }
}
